_episodes/41.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: episode
+title: "41: Trino puts on its Hudi"
+date: 2022-10-27
+tags: hudi apache datalake table format sql
```

Comment:
lakehouse ?

Reply:
I've traditionally used datalake for these connectors

Reply:
ok .. we might want to add lakehouse at some stage soon